pipeline/scripts/ci/cos_cli.py

```diff
@@ -0,0 +1,137 @@
+"""This script is to update report artifacts on IBM
+
+- Upload test result files to "qe-ci-reports" bucket
+- Executes bucket file options like upload, download, delete and list
+"""
+
+import logging
+import sys
+
+from docopt import docopt
+
+from storage.ibm_cos import CloudObjectStorage
+
+LOG = logging.getLogger(__name__)
+
+usage = """
+This script helps to update reports in IBM.
+
+Usage:
+  manage_reports.py (upload|download) <bucket_name> (-f FILE_PATH) (-k OBJ_KEYNAME)
+  manage_reports.py delete <bucket_name> (-k OBJ_KEYNAME)
+  manage_reports.py list <bucket_name>
+  manage_reports.py -h | --help
+
+    action              upload | download | delete | list-objects
+    bucket_name         bucket name
+
+Example:
+    python manage_reports.py upload qe-ci-reports test.xml test-run-1
+    python manage_reports.py download qe-ci-reports test.xml test-run-1
+    python manage_reports.py delete qe-ci-reports test-run-1
+    python manage_reports.py list qe-ci-reports
```

Comment:
Can we update the description in accordance with the filename modification?

Another request, would it make sense if we do
```
# cos_cli upload <source_file> <bucket_name> 
# cos_cli download <object_name> <destination> --bucket-name <name of bucket>
# cos_cli delete <object_name> --bucket-name <name of bucket>
# cos_cli list --bucket-name <name of bucket>
```